### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by sjinks**
Update CHANGELOG.md and amend package.json.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -4,63 +4,7 @@ Release tasks:
 
 * `npm run lint` - fix any linting issues.
 * `npm run test` - fix any failing tests.
-* Test that it works with (or prepare the necessary changes to make it work with):
-    * [eslint-eslines](https://github.com/Automattic/eslint-eslines)
```
Comment (sjinks) [inline RELEASE.md:8]: Works only with eslines 1.0

File: RELEASE.md [hunk 2]
```diff
@@ -4,63 +4,7 @@ Release tasks:
...
-    * major contributors: [wp-calypso](https://github.com/Automattic/wp-calypso), Happychat ([service](https://github.com/Automattic/happychat-service) et al.)
```
Comment (sjinks) [inline RELEASE.md:9]: Happychat is closed, Calypso dropped eslines in 2019

File: package.json [hunk 3]
```diff
@@ -42,5 +42,9 @@
   },
   "peerDependencies": {
     "eslint": ">=6.0.0"
-  }
+  },
+  "files": [
+    "src/**/*.js",
+    "index.js"
+  ]
```
Comment (sjinks) [inline package.json:49]: Publish only what is necessary

**Review by pschoffer — APPROVED** [review]
LGTM

**Pull request merged**